### PR TITLE
Add unifi device tracker documentation

### PR DESCRIPTION
**Pull request by kk7ds**

### CODE REVIEW

File: source/_components/device_tracker.unifi.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+layout: page
+title: "Ubiquiti Unifi WAP"
+description: "Instructions how to use a Unifi WAP controller as a device tracker module."
+date: 2016-02-19 20:59
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ubiquiti.png
+ha_category: Presence Detection
```
Comment (balloob) [inline source/_components/device_tracker.unifi.markdown]: Could you add:
```yaml
ha_iot_class: "Local Poll"
```

**Comment by balloob** [comment]
You can merge it after the comment is addressed.

File: source/_components/device_tracker.unifi.markdown (hunk 1 above)
Comment (kk7ds) [inline source/_components/device_tracker.unifi.markdown]: Should this be "Local Polling"?

https://github.com/balloob/home-assistant.io/blob/master/source/_components/light.hue.markdown

I guess I missed the start of the tagging effort, sorry about that. Excellent idea though.

Comment (balloob) [inline source/_components/device_tracker.unifi.markdown]: Oops you're right. 

The tags are not being used yet but I plan to use it to highlight the more useful components on the site

Comment (happyleavesaoc) [inline source/_components/device_tracker.unifi.markdown]: "Useful" is pretty subjective. "Accurate" might be a better descriptor.

**Pull request merged**